snap/snapcraft.yaml

```diff
@@ -19,39 +19,101 @@ slots:
-      - password-manager-service
+      - removable-media
 
 parts:
+  ffmpeg:
+    # WORKAROUND:
+    # Build from source because the ffmpeg package installs libraries as dependencies
+    # that conflict with the Gnome extension
+    plugin: autotools
+    source: https://ffmpeg.org/releases/ffmpeg-6.0.tar.xz
+    source-checksum: sha256/57be87c22d9b49c112b6d24bc67d42508660e6b718b3db89c44e47e289137082
+    autotools-configure-parameters:
+      # WORKAROUND: Install to /usr instead of /usr/local because it's not in search paths
+      - --prefix=/usr
+      - --disable-debug
+      - --disable-doc
+      - --disable-static
+      - --enable-gpl
+      - --enable-shared
+      - --disable-ffplay
+      - --disable-devices
+      - --enable-gnutls
+      - --enable-libmp3lame
+      - --enable-libvorbis
+    build-packages:
+      - nasm
+      - libgnutls28-dev
+      - libmp3lame-dev
+      - libvorbis-dev
+    stage-packages:
+      - libmp3lame0
+    stage:
+      - -usr/include
+  gtksourceview:
+    source: https://gitlab.gnome.org/GNOME/gtksourceview.git
+    source-branch: 'gtksourceview-4-6'
+    source-depth: 1
+    plugin: meson
+    meson-parameters:
+     - --prefix=/usr
+     - -Ddebug=true
+     - -Doptimization=3
+    build-packages:
+      - gettext
+      - libxml2-dev
+    override-pull: |
+      set -eux
+      snapcraftctl pull
+      sed -i 's#Werror=missing-include-dirs#Wmissing-include-dirs#g' meson.build
+    prime:
+      - usr/lib/*/libgtksourceview*
+  rustup:
+    plugin: nil
+    build-packages: [wget]
+    build-environment:
+      - RUSTUP_HOME: $CRAFT_PART_INSTALL/usr/share/rust
+      - CARGO_HOME: $CRAFT_PART_INSTALL/usr/share/rust
+      - CARGO_BUILD_JOBS: $CRAFT_PARALLEL_BUILD_COUNT
+    override-pull: |
+      wget https://sh.rustup.rs -O $CRAFT_PART_SRC/rustup-init.sh
+      chmod +x $CRAFT_PART_SRC/rustup-init.sh
+    override-build: |
+      $CRAFT_PART_SRC/rustup-init.sh -y --no-modify-path
+      mkdir -p $CRAFT_PART_INSTALL/usr/bin
+      for i in `ls $RUSTUP_HOME/bin/`; do
+        ln -s ../share/rust/bin/$i $CRAFT_PART_INSTALL/usr/bin/$i
+      done
+    override-prime: |
+      echo 'Skip Prime'
   fractal:
+    after: [rustup, gtksourceview, ffmpeg]
     parse-info: [usr/share/metainfo/org.gnome.Fractal.metainfo.xml]
     source: https://gitlab.gnome.org/GNOME/fractal.git
     source-type: git
-    source-commit: 8781378a797d10d6b8e22698aa7cc8d825f13ae5
+    source-tag: $SNAPCRAFT_PROJECT_VERSION
     plugin: meson
-    meson-parameters: [--prefix=/snap/fractal/current/usr]
+    meson-parameters: [--prefix=/snap/fractal/current/usr, -Dprofile=default]
     organize:
       snap/fractal/current/usr: usr
     build-packages:
-      - appstream
       - curl
       - libssl-dev
       - libgstreamer1.0-dev
       - libgstreamer-plugins-bad1.0-dev
       - libgstreamer-plugins-base1.0-dev
       - libgstreamer-plugins-good1.0-dev
       - libges-1.0-dev
-      - cmake
+      - libhandy-0.0-dev
```

Comment:
libhandy-1 is in the sdk snap... perhaps fractal still needs the older one, just double check what is really needed

Reply:
fractal needs libhandy-0 and libhandy-1 is in gnome-sdk. I'll build the latest master branch today.